archetypes/database-mp/src/main/resources/pom.xml.mustache

```diff
@@ -43,6 +43,14 @@
             <groupId>jakarta.transaction</groupId>
             <artifactId>jakarta.transaction-api</artifactId>
         </dependency>
+        <!-- dependency>
+            <groupId>io.helidon.integrations.db</groupId>
+            <artifactId>ojdbc</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.helidon.integrations.db</groupId>
+            <artifactId>h2</artifactId>
+        </dependency -->
         <dependency>
             <groupId>com.h2database</groupId>
```

Comment:
If you use the dependency above, this one is not needed (and you do not need to comment out anything for native image)